exercises/calculator.cpp

```diff
@@ -10,3 +10,24 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+
+using namespace std;
+
+int main(int argc, char const *argv[])
+{
+  int x,y=0; 
+
+  cout<<"insert two numbers"<<endl;  
+  //take input
+  cin>>x; 
+  cin>>y; 
+  //print output
+  cout<<"sum "<<x+y<<endl;
+  cout<<"difference "<<x-y<<endl;
+  cout<<"Multiplication "<<x*y<<endl;
+  cout<<"Division "<<x/y<<endl;
```

Comment:
dovresti mettere più spazi, ad esempio:
```suggestion
  cout << "Division " << x/y << endl;
```

Reply:
inoltre,potresti creare una funzione per ogni operazione